packages/react/src/isomorphicClerk.ts

```diff
@@ -574,27 +488,27 @@ export class IsomorphicClerk implements IsomorphicLoadedClerk {
       clerkjs.openWaitlist(this.preOpenWaitlist);
     }
 
-    this.premountSignInNodes.forEach((props: SignInProps, node: HTMLDivElement) => {
+    this.premountSignInNodes.forEach((props, node) => {
```

Comment:
I've also removed unnecessary type annotations and instead am relying on the underlying `IsomorphicClerk` type definition.